### PR TITLE
docs: remove mentions of nonexistent next api routes

### LINKED ISSUE #1127: bug(docs): folder structure page mentions nonexistent next api routes
### Describe the bug

The folder structures page (https://create.t3.gg/en/folder-structure#srcpagesapi, and presumably same for other languages) mentions some files we don't have anymore:

> The examples.ts file contains an example of a route that uses the [Next.js API route↗](https://nextjs.org/docs/api-routes/introduction) feature along with Prisma. The restricted.ts file contains an example of a route that uses the [Next.js API route↗](https://nextjs.org/docs/api-routes/introduction) feature and is protected by [NextAuth.js↗](https://next-auth.js.org/).

I think we should still keep the `src/pages/api` heading with the first sentence that describes what it does. Could also add another sentence or two if someone thinks there's something else worth mentioning.

### PR DESCRIPTION
**Pull request by ronthetech**
Closes #1127

## ✅ Checklist

- ✅ I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- ✅ The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- ✅ I performed a functional test on my final commit

---

## Changelog

removed two lines that mention nonexistent next api routes

---

## Screenshots

before
after

💯

### CODE REVIEW

File: www/src/pages/en/folder-structure.mdx [hunk 1]
```diff
@@ -47,7 +47,7 @@ The `pages` folder contains all the pages of the Next.js application. The `index
 
 #### `src/pages/api`
 
-The `api` folder contains all the API routes of the Next.js application. The `examples.ts` file contains an example of a route that uses the [Next.js API route](https://nextjs.org/docs/api-routes/introduction) feature along with Prisma. The `restricted.ts` file contains an example of a route that uses the [Next.js API route](https://nextjs.org/docs/api-routes/introduction) feature and is protected by [NextAuth.js](https://next-auth.js.org/).
+The `api` folder contains all the API routes of the Next.js application.
```
Comment (juliusmarminge) [inline www/src/pages/en/folder-structure.mdx:50]: I think we should put a few more words here given that it's one of Next.js' strong suits

Comment (c-ehrlich) [reply www/src/pages/en/folder-structure.mdx:50]: linking here probably makes sense https://nextjs.org/docs/api-routes/introduction

Comment (ronthetech) [reply www/src/pages/en/folder-structure.mdx]: I added the link to the Next.js api routes docs, thank you for the suggestion!

**Review by c-ehrlich — APPROVED** [review]
looks good, thanks for catching this :)

**Comment by ronthetech** [comment]
Thanks for making it easy to contribute!

**Pull request merged**